11LessonEleven/in-class-project/index.html

```diff
@@ -1,33 +1,37 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title>Lesson Eleven - In Class</title>
-  <link rel="stylesheet" href="css/style.css">
-  <link rel="stylesheet" media="(max-width: 800px)" href="css/style.css" />
+   <title>Lesson Eleven - In Class</title>
+   <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-  <section class="hero" "clear-fix">
-    <h1>Welcome to our <strong>site</strong></h1>
-  </section>
-  <footer class="clear-fix">
-    <div class="foot1 one">
-      <h1>Title</h1>
-      <p>
-        Bacon ipsum dolor amet beef ribs meatball alcatra, tri-tip andouille shankle leberkas prosciutto strip steak jowl.
-      </p>
-    </div>
-    <div class="foot1 two">
-      <h1>Title</h1>
-      <p>
-        Bacon ipsum dolor amet beef ribs meatball alcatra, tri-tip andouille shankle leberkas prosciutto strip steak jowl.
-      </p>
-    </div>
-    <div class="foot1 three">
-      <h1>Title</h1>
-      <p>
-        Bacon ipsum dolor amet beef ribs meatball alcatra, tri-tip andouille shankle leberkas prosciutto strip steak jowl.
-      </p>
-    </div>
-  </footer>
+   <header>
+       <h1>Welcome to our <b>site</b></h1>
+   </header>
+   <main class="clearfix">
+       <div class="box one">
+           <h2>Title</h2>
+           <p>
+               In microdosing irony, pabst kale chips church-key seitan sint. Est occupy readymade gastropub, flannel lomo waistcoat
+           </p>
+       </div>
+       <div class="box two">
+           <h2>Title</h2>
+           <p>
+               In microdosing irony, pabst kale chips church-key seitan sint. Est occupy readymade gastropub, flannel lomo waistcoat
+           </p>
+       </div>
+       <div class="box three">
+           <h2>Title</h2>
+           <p>
+               In microdosing irony, pabst kale chips church-key seitan sint. Est occupy readymade gastropub, flannel lomo waistcoat
+           </p>
+       </div>
+   </main>
 </body>
+
 </html>
```

Comment:
Great job. It looks like when I go to the phone size, there's a black box still hanging out on the right. Please remove this and I can bump your grade up.

<img width="430" alt="screen shot 2016-09-13 at 5 47 25 pm" src="https://cloud.githubusercontent.com/assets/12698279/18494186/4bc97106-79da-11e6-8af8-7b52938393ab.png">